website/docs/components/outputs/gcp_gcs.md

```diff
@@ -0,0 +1,300 @@
+---
+title: gcp_gcs
+type: output
+status: stable
+categories: ["Services","GCP"]
+---
+
+<!--
+     THIS FILE IS AUTOGENERATED!
+
+     To make changes please edit the contents of:
+     lib/output/gcp_gcs.go
+-->
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+Sends message parts as objects to a Google Cloud Storage bucket. Each object is
+uploaded with the path specified with the `path` field.
+
+Introduced in version 3.41.1.
+
+
+<Tabs defaultValue="common" values={[
+  { label: 'Common', value: 'common', },
+  { label: 'Advanced', value: 'advanced', },
+]}>
+
+<TabItem value="common">
+
+```yaml
+# Common config fields, showing default values
+output:
+  gcp_gcs:
+    bucket: ""
+    path: ${!count("files")}-${!timestamp_unix_nano()}.txt
+    content_type: application/octet-stream
+    max_in_flight: 1
+    batching:
+      count: 0
+      byte_size: 0
+      period: ""
+      check: ""
+```
+
+</TabItem>
+<TabItem value="advanced">
+
+```yaml
+# All config fields, showing default values
+output:
+  gcp_gcs:
+    bucket: ""
+    path: ${!count("files")}-${!timestamp_unix_nano()}.txt
+    content_type: application/octet-stream
+    content_encoding: ""
+    chunk_size: 16777216
```

Comment:
This might trip some users, but it's the default value of `googleapi.DefaultUploadChunkSize` and it's an optional feature of the Google storage writer which can increase / decrease performance in some cases, depending on the message size. Might be useful for some people, but not sure. I can leave it out of the config if you think it's too confusing.

Reply:
I think this is fine, I'll probably mark this as an experimental component at first so we'll have an opportunity to get feedback.